types/cabarc/index.d.ts

```diff
@@ -0,0 +1,151 @@
+// Type definitions for cabarc 0.4
+// Project: https://github.com/jhermsmeier/node-cabarc
+// Definitions by: Zlatko Andonovski <https://github.com/Goldsmith42>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.6
```

Comment:
If you write get/set props:
```ts
get readOnly(): number;
set readOnly(value: number);
```
into:
```ts
readOnly: number;
```
(these are regular get/set, no readonly feature), this will remove requirements for TS 3.6, what do you think?

Reply:
That makes sense. I was trying to mirror the actual code, but doing it this way won't affect type checking in any meaningful way.

Reply:
> That makes sense. I was trying to mirror the actual code, but doing it this way won't affect type checking in any meaningful way.

yes, win-win, thx!